### PR TITLE
hw2-solution-task lottery

### PR DESCRIPTION
**Pull request by inshkliar**

### CODE REVIEW

File: src/main/java/core/basesyntax/Application.java [hunk 1]
```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
 public class Application {
+    private static int COUNT_OF_BALL = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lotteries = new Lottery[COUNT_OF_BALL];
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Application.java:7]: We do not need many lotteries. Same as we do not need 3 shops to buy 3 milk bottles.

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+class ColorSupplier {
+    private Random random = new Random();
+
+    String getRandomColor() {
+        int indexOfColor = random.nextInt(Colors.values().length);
+        return Colors.values()[indexOfColor].toString();
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/ColorSupplier.java:10]: image

File: src/main/java/core/basesyntax/Colors.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Colors.java:3]:
```suggestion
public enum Color {
```

**Review by Rommelua — CHANGES_REQUESTED** [review]
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery
Before you create PR with your homework, you need to go through the checklist under the task and correct all the points described there. The mentor will not check the work until the checklist points are corrected

**Review by aaananas — APPROVED** [review]

**Pull request closed without merging**